app/models/setting/ansible.rb

```diff
@@ -106,6 +106,12 @@ def load_defaults
                  'foreman_params["host_parameter"] in the playbooks.'),
               true,
               N_('Top level Ansible variables')
+            ),
+	    set(
+              'ansible_interval',
+              N_('Timeout (in minutes) when hosts should have reported.'),
```

Comment:
"0 if you do not require hosts to report via Ansible on a fixed interval", also 0 by default